### ISSUE TITLE
Cloning non-existent repositories

### ISSUE THREAD

**Issue by juanfradb** [Significant-Gravitas/Auto-GPT#2717]
### ⚠️ Search for existing issues first ⚠️

- [X] I have searched the existing issues, and there is no existing issue for my problem

### Which Operating System are you using?

Windows

### GPT-3 or GPT-4?

GPT-3.5

### Current behavior 😯

NEXT ACTION:  COMMAND = clone_repository ARGUMENTS = {'repository_url': 'https://github.com/(invented author)/(invented project).git', 'clone_path': './transcribed_text'}

Basically autogpt try to clone repositories that doesnt exists.

### Your prompt 📝

```yaml
# Paste your prompt here
```

### Your Logs 📒

```log
<insert your logs here>
```

**Comment by ntindle** [comment]
Limitation of AI Model
